CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Added
 
 - Wallet address contract invocation input validation for payable contracts.
+- Add `network_id` to `WalletData` so that it is saved with the seed data and surfaced via the export function
```

Comment:
Please add new changes under `Unreleased` heading until the new release is being cut